app/bundles/EmailBundle/Views/Send/form.html.twig

```diff
@@ -0,0 +1,45 @@
+{% extends 'MauticCoreBundle:Default:content.html.twig' %}
+
+{% block mauticContent 'emailSend' %}
+
+{% block headerTitle 'mautic.email.send.list'|trans({'%name%': email.name})|raw %}
```

Comment:
Same here. Can we use `purify`? It prints user-defined input so could be harmful.

Reply:
https://github.com/mautic/mautic/pull/11974/commits/e66dea9e6d8d3815c67af4d83ff0e8633e21f6a9

Route: `/s/emails/send/{id}`